src/main/java/seedu/jarvis/logic/parser/TaskCommandParser.java

```diff
@@ -140,7 +140,7 @@ public static LocalDateTime parseTimedTaskTime(String[] nameKeywords, int length
 
         } else if (!hasDatePrefix || !hasTimePrefix) {
             throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_MISSING_DATE));
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_MISSING_INFO));
```

Comment:
I think this error message is incorrect, I might still need the previous version error message.

Reply:
Okay, just a sec!

Reply:
Just pushed a new commit, hopefully it should work now!

Reply:
THank you! 👍